tgui/packages/tgui/interfaces/GasMixer/MixInfo.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Garash2k (https://github.com/garash2k)
+ * @license MIT
+ */
+
+import { Box, LabeledList, NoticeBox } from 'tgui-core/components';
+
+import { formatPressure } from '../../format';
+import type { AirInfo } from './types';
+
+interface MixInfoProps {
+  mix: AirInfo;
+}
+export const MixInfo = (props: MixInfoProps) => {
+  const { mix } = props;
+  return (
+    <LabeledList>
+      {mix.kpa && mix.temp ? (
```

Comment:
Below you output `mix.temp` in degrees centigrade. Here you have a truthy check on `mix.temp`. `0` is falsey, so if `mix.temp` is 0C this won't show.